docs/user-guide/attributes-table.md

```diff
@@ -78,11 +78,15 @@ The user is always allowed to delete the drawn shape through the <img src="../im
 
 ### Editing and removing existing features
 
-In order to update an existing feature, it is necessary to start the *Editing session* by clicking the *Edit mode* icon <img src="../img/button/editing-button.jpg" class="ms-docbutton"/> button. If the goal is to update the Attribute Table records, the user can simply select them and type the desired text into the input field. However, it is also possible to modify the geometries associated with a record by moving the existent vertices and/or creating new ones directly from the map.
+In order to update an existing features, it is necessary to start the *Editing session* by clicking the *Edit mode* icon <img src="../img/button/editing-button.jpg" class="ms-docbutton"/> button. If the goal is to update the Attribute Table records, the user can simply select them and type the desired text into the input field. However, it is also possible to modify the geometries associated with a record by moving the existent vertices and/or creating new ones directly from the map.
 
 <img src="../img/attributes-table/update-polygon-shape.gif" class="ms-docimage"/>
 
-With a click on **Save changes** <img src="../img/button/save-changes.jpg" class="ms-docbutton"/> these changes will be persistent.<br> In order to delete an existing feature, the user can always, in *Edit mode*, select that feature and click on the <img src="../img/button/delete-features-button.jpg" class="ms-docbutton" /> button.
+!!! note
+    To change existing features by changing multiple cells with the same content, you can drag/copy a cell value to multiple cells, as follows:
```

Comment:
It is possible to edit the value of an attribute for multiple records at once by selecting the corresponding cell in the table and dragging the content onto the multiple cells, as follows:
    <img src="../img/attributes-table/multiple_features.gif" class="ms-docimage" style="max-width:300px;"/>